src/apps/common/submodules/conferenceViewer/conferenceViewer.js

```diff
@@ -374,6 +375,9 @@ define(function(require) {
 
 					break;
 				case 'start-talking':
+					if ($userDiv.find('[data-action-category="mute"]').hasClass('active')) {
+						break;
```

Comment:
and replace your additional `break` statement by adding the class instead:
```suggestion
						$userDiv.addClass('speaking');
```